manifests/quick-start-minimal.yaml

```diff
@@ -1396,6 +1396,10 @@ data:
       secretKeySecret:
         name: my-minio-cred
         key: secretkey
+  columns: |
+    - name: Workflow Completed
+      type: label
+      key: workflows.argoproj.io/completed
```

Comment:
I wonder if users would also want this column to include workflows that match multiple label keys. Would it be better for this to be a label selector?